ocean_py/agent/purchase_agent.py

```diff
@@ -0,0 +1,28 @@
+"""
+    PublishAgent - Agent to access the ocean publishing and services aka Brizo
+"""
+
+from squid_py import ServiceAgreement, ServiceTypes
+
+from ocean_py.agent.agent_base import AgentBase
+# from ocean_py import logger
+
+class PurchaseAgent(AgentBase):
+    def __init__(self, ocean, **kwargs):
+        """init a standard ocean agent, with a given DID"""
+        AgentBase.__init__(self, ocean, **kwargs)
+
+
+    def purchase_asset(self, asset, account):
+        """
+        Register an asset with the agent storage server
+        :param asset: asset to purchase
```

Comment:
Should the comment call out what type 'asset' must be?

Reply:
not exactly sure at the moment, but in theory if it's an already purchased asset, then squid will go through the same steps and create a service level agreement, and go through the purchasing steps as normal